static/app/views/alerts/rules/metric/triggers/chart/index.tsx

```diff
@@ -632,6 +636,17 @@ class TriggersChart extends PureComponent<Props, State> {
             {noop}
           </EventsRequest>
         ) : null}
+        {this.props.includeConfidence ? (
+          <EventsRequest
+            {...baseProps}
+            api={this.confidenceAPI}
+            yAxis="count(span.duration)"
```

Comment:
hmm, should the confidence be based on the selected aggregate? what if the user wants to alert on p75(span.duration)?

Reply:
hmm, would `count(span.duration)` produce different confidence from `p75(span.duration)`? I could be understanding wrong, but I thought it might be the same because it's the size of the selected group that matters and not the aggregation. 

But I think your point is still valid because the user might select a different "metric" instead of `span.duration` which could definitely affect confidence. I'll update to use the selected aggregate as you suggested!